test/Kongverge.IntegrationTests/ProgramInvalidArgumentsScenarios.cs

```diff
@@ -49,5 +49,16 @@ public class ProgramInvalidArgumentsScenarios : ProgramSteps
                 .When(x => x.InvokingMain())
                 .Then(x => x.TheExitCodeIs(ExitCode.IncompatibleArguments))
                 .BDDfy();
-    }
+
+        [BddfyFact(DisplayName = nameof(AValidUser))]
```

Comment:
This `DisplayName` should fully describe all the `Given` steps, like the tests above do.